beeline-examples/package.yaml

```diff
@@ -21,37 +21,59 @@ dependencies:
 - beeline-routing >= 0.2.1 && < 0.3
 
 flags:
-  strict:
+  ci:
     description: More strict ghc options used for development and ci, not intended for end-use.
     manual: true
     default: false
 
 when:
-  - condition: flag(strict)
+  - condition: flag(ci)
     then:
       ghc-options:
-        - -Weverything
-        - -Werror
-        - -Wno-missing-local-signatures
-        - -Wno-monomorphism-restriction
-        - -Wno-missing-kind-signatures
-        - -Wno-prepositive-qualified-module
-        - -Wno-implicit-prelude
-        - -Wno-safe
-        - -Wno-unsafe
-        - -Wno-missing-safe-haskell-mode
-        - -Wno-missing-deriving-strategies
-        - -Wno-all-missed-specialisations
-        - -Wno-missed-specialisations
-        - -Wno-unticked-promoted-constructors
+          - -O2
+          - -Wall
+          - -Werror
+          - -Wcompat
+          - -Widentities
+          - -Wmissing-local-signatures
+          - -Wmissing-export-lists
+          - -Wpartial-fields
+          - -Wmissed-specialisations
+          - -Wunused-packages
+          - -Wno-implicit-prelude
+          - -Wno-safe
+          - -Wno-unsafe
+      when:
+          - condition: impl (ghc >= 9.0)
+            ghc-options:
+              - -Winvalid-haddock
+          - condition: impl (ghc <= 9.3)
+            #Previously the guidance was to always used ticked constructors for type constructors,
+            #but that guidance has been reversed, so we turn the warning off in older compilers.
+            ghc-options:
+              - -Wno-unticked-promoted-constructors
+
+          - condition: impl (ghc >= 9.0)
+            ghc-options:
+              - -Winvalid-haddock
```

Comment:
Possibly this is the place where the duplication should be removed

Reply:
Yeah I missed updating the beeling-examples/package.yaml, fixed now. Thanks!